docs/tutorial/index.md

```diff
@@ -282,32 +346,38 @@ To enable high availability in a production environment, replicas should be host
 
 ### Add units
 
-Currently, your deployment has only one juju **unit**, known in juju as the **leader unit**. You can think of this as the database **primary instance**. For each **replica**, a new unit is created. All units are members of the same database cluster.
+Currently, your deployment has only one Juju unit, known in juju as the **leader unit**. You can think of this as the database's **primary instance**. For each cluster replica, a new Juju unit is created. 
```

Comment:
This is somewhat misleading, since Juju and Patroni track leadership separately. When deploying, they will match, but may drift apart over time.